ea_restaurant_data_loader/test/fixtures/app_client_fixture.ex

```diff
@@ -0,0 +1,28 @@
+defmodule EaRestaurantDataLoader.AppClientFixture do
+  alias EaRestaurantDataLoader.AppClient
+  alias EaRestaurantDataLoader.Repo
+
+  def build_app_client(client_name, client_id, user),
+    do:
+      %AppClient{}
+      |> AppClient.changeset(%{
+        client_name: client_name,
+        client_id: client_id,
+        client_secret: "secret",
+        access_token_expiration_time: 10,
+        refresh_token_expiration_time: 100,
+        entity_status: "ACTIVE",
+        created_by: user.id,
+        updated_by: user.id,
+        created_date: DateTime.utc_now(),
+        updated_date: DateTime.utc_now()
+      })
+
+  def build_and_insert_app_client(client_name, client_id, user) do
+    {:ok, inserted_app_client} =
```

Comment:
Return the whole insert response and do the validation in the test if the insertion was :ok

Reply:
Done.